projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -396,6 +396,34 @@ public function add_options_menu() {
 
 		$this->update_submenus( 'options-general.php', $submenus_to_update );
 
+		if (
+			/**
+			 * Filter to enable the Newsletter Settings section in Calypso UI.
+			 *
+			 * @since $$next-version$$
+			 * @module masterbar
+			 *
+			 * @param bool false Enable newsletter setting section? Default to false.
+			 */
+			apply_filters( 'calypso_use_newsletter_settings', '__return_false' )
```

Comment:
Are the filters in there to offer more control as to when you'll enable those menu items?

Reply:
Correct, since the UIs in calypso are in different PRs (links in the description) and deploying Jetpack PRs on .com isn't super straightforward/fast, I wanted to have easier time enabling things freely on .com when I wanted.